src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -193,7 +188,7 @@ export function getMostRecentBundleBlockRanges(
     const hubPoolStartBlock = getBlockRangeForChain(
       rootBundleBlockRanges,
       hubPoolClient.chainId,
-      hubPoolClient.configStoreClient.enabledChainIds
+      hubPoolClient.configStoreClient.getEnabledChains(latestExecutedRootBundle.blockNumber)
```

Comment:
```suggestion
      hubPoolClient.configStoreClient.getEnabledChains()
```

I don't know how you arrived at passing in `latestExecutedRootBundle.blockNumber` as the block here but we should just use the latest chain ID list because the list can only get longer

Reply:
Right, my understanding was that we wanted to get as close to the start of the bundle as possible. But yes, since we're using an append-only list, then this should be gtg with the additional commit